scripts/iter_command.ps1

```diff
@@ -0,0 +1,34 @@
+
+# For local unit testing, often certain bugs only appear due to a specific ordering of events in the network.
```

Comment:
Can we instead integrate this into `test.ps1` directly?

Reply:
I'll look into it!